docs: refresh Agent Swarm TUI guide (#638)
VRSEN/agency-swarm: docs/core-framework/agencies/agent-swarm-cli.mdx

Once the TUI is running, you get:

- a keyboard-first terminal workflow

> - session history, export, and `/compact`

Author: undo and redo should be included, revert the line

> - `/agents` to switch between the agency and its top-level agents

Author: replace agency with swarm
```suggestion
- `/agents` to switch between the swarms and their agents
```

- `@AgentName` mentions to route a prompt to a specific top-level agent

> - `@` file and resource references when local project context is available

Author: Refine based on facts.

- `@AgentName` mentions to route a prompt to a specific top-level agent
- `@` file and resource references to attach local or MCP context in the same prompt flow
- `@` file and resource references when local project context is available
- `Tab` autocomplete for both agent mentions and file and resource references

> - direct access to local project files for prompt context

Author: You can attach files. Ensure this page is fully aligned with the actual package functionality.
